src/main/java/org/gridsuite/modification/server/entities/equipment/creation/LineCreationEntity.java

```diff
@@ -81,6 +78,16 @@ public LineCreationEntity(String equipmentId,
         this.shuntSusceptance2 = shuntSusceptance2;
     }
 
+    @Override
+    public void update(ModificationInfos modificationInfos) {
+        super.update(modificationInfos);
+        LineCreationInfos lineCreationInfos = (LineCreationInfos) modificationInfos;
+        shuntConductance1 = lineCreationInfos.getShuntConductance1();
+        shuntSusceptance1 = lineCreationInfos.getShuntSusceptance1();
+        shuntConductance2 = lineCreationInfos.getShuntConductance2();
+        shuntSusceptance2 = lineCreationInfos.getShuntSusceptance2();
+    }
+
     public static LineCreationEntity toEntity(LineCreationInfos lineCreationInfos) {
```

Comment:
Use in the constructor 
```
                currentLimit1 != null ? currentLimit1.toEntity() : null,
                currentLimit2 != null ? currentLimit2.toEntity() : null,
```
and change it to private

Reply:
Done